### PR TITLE
Update osfiber_windows.h

### PR DESCRIPTION
**Pull request by a952135763**
I think you should use CreateFiberEx Instead of CreateFiber

### CODE REVIEW

**Comment by a952135763** [comment]
@googlebot I signed it!

File: src/osfiber_windows.h [hunk 1]
```diff
@@ -64,10 +64,12 @@ OSFiber::~OSFiber() {
 Allocator::unique_ptr<OSFiber> OSFiber::createFiberFromCurrentThread(
     Allocator* allocator) {
   auto out = allocator->make_unique<OSFiber>();
-  out->fiber = ConvertThreadToFiber(nullptr);
+  // Windows XP:  The FIBER_FLAG_FLOAT_SWITCH flag is not supported.
+  // ConvertThreadToFiber -->> ConvertThreadToFiberEx
```
Comment (ben-clayton) [inline src/osfiber_windows.h:68]: `// ConvertThreadToFiber -->> ConvertThreadToFiberEx`

This line is a bit meaningless. You've described a change that can be found in the git history.
I'd suggest removing this line.

File: src/osfiber_windows.h [hunk 2]
```diff
@@ -77,9 +79,12 @@ Allocator::unique_ptr<OSFiber> OSFiber::createFiber(
     size_t stackSize,
     const std::function<void()>& func) {
   auto out = allocator->make_unique<OSFiber>();
-  out->fiber = CreateFiber(stackSize, &OSFiber::run, out.get());
+  // Windows XP:  The FIBER_FLAG_FLOAT_SWITCH flag is not supported.
+  // CreateFiber -->> CreateFiberEx
```
Comment (ben-clayton) [inline src/osfiber_windows.h:83]: Again, I'd remove this line.

File: src/osfiber_windows.h [hunk 3]
```diff
@@ -77,9 +79,12 @@ Allocator::unique_ptr<OSFiber> OSFiber::createFiber(
...
+  // least stackSize 64KB
```
Comment (ben-clayton) [inline src/osfiber_windows.h:84]: According [to the docs](https://docs.microsoft.com/en-us/windows/win32/procthread/thread-stack-size):

> The operating system rounds up the specified size to the nearest multiple of the system's allocation granularity (typically 64 KB).

So maybe this should be:

`// stackSize is rounded up to the system's allocation granularity (typically 64 KB).`

**Comment by a952135763** [comment]
This is one of the reasons.
The CreateFiber does not preserve floating-point state on x86.
The CreateFiber minimum StackSize is 1Mb.

**Comment by ben-clayton** [comment]
> The CreateFiber does not preserve floating-point state on x86.
> The CreateFiber minimum StackSize is 1Mb(Cannot be smaller).

Great, thanks for the clarification. Please can you add these lines to the CL description?

**Comment by ben-clayton** [comment]
Perfect, thank you!

**Pull request merged**